### PR TITLE
New report page

### PR DESCRIPTION
**Pull request by laylapku**
Report page update based on latest design on Zeplin

### CODE REVIEW

File: src/components/banner/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  background-color: ${color.extraLightPurple};
+  color: ${color.primary};
+  align-self: flex-start;
+
+  @media (min-width: 60rem) {
+    width: ${({ width }) => (width ? width : "auto")};
+  }
+`;
+
+const Banner = ({ text, width }) => {
```
Comment (egor-sorokin) [inline src/components/banner/Banner.jsx:22]: Not that crusial, but you can write it without return, like so
```
const Banner = ({ text, width }) => (
<BannerWrapper width={width}>{text}</BannerWrapper>;
);
```

File: src/components/details/Details.jsx [hunk 2]
```diff
@@ -75,9 +65,9 @@ const Details = ({ children, title }) => {
     <DetailsContainer onToggle={handleOpen}>
       <StyledSummary>
         {title}
-        <IconWrapper>
+        <div>
```
Comment (egor-sorokin) [inline src/components/details/Details.jsx:68]: same question as in another PR, do you really need this div or you can get rid of it

File: src/data/reportData.js [hunk 3]
```diff
@@ -4,60 +4,93 @@ export const mockReportData = {
...
             "background color: #f2f2f2",
             "font size: 45pt",
             "font weight: bold",
           ],
-          link: "/",
+          affectedElements: [
+            {
+              page: "https://futurice.com/careers/lead-developer-helsinki",
+              selectors: [
+                ".css-1e95rm6.eh1sm5o1:nth-child(1) > a > .css-1favn2n.eh1sm5o2 > .css-t5irqm.eh1sm5o3",
```
Comment (egor-sorokin) [inline src/data/reportData.js:65]: Is that mocked data? Should be removed after backend is done? If so, could you please add todo in the beginning that says so.

File: src/components/report/OverviewCard.jsx [hunk 4]
```diff
@@ -31,28 +30,15 @@ const Title = styled.div`
...
-
-const GuidelineContainer = styled.div`
+const IssueContainer = styled.div`
   display: flex;
   flex-wrap: wrap;
-
-  a {
-    flex: 1 0 40%;
-    padding: 0.5rem;
+  & > * {
```
Comment (egor-sorokin) [inline src/components/report/OverviewCard.jsx:36]: Do you really need to use * here? mb there is a common tag like div or whatever and you can use it

File: src/components/banner/Banner.jsx (hunk 1 above)
Comment (laylapku) [reply src/components/banner/Banner.jsx:22]: You are absolutely right. Just removed it.

File: src/data/reportData.js (hunk 3 above)
Comment (laylapku) [reply src/data/reportData.js:65]: Yes, I added a todo card for this on Trello.

File: src/components/report/OverviewCard.jsx (hunk 4 above)
Comment (laylapku) [reply src/components/report/OverviewCard.jsx]: The only child elements of this container are identical Badge components which are wrapped in divs. I changed `*` to `div` to make it more specific.

**Pull request merged**